lightkube/core/internal_models.py

```diff
@@ -2,6 +2,11 @@
 
 try:
     from ..models import meta_v1, autoscaling_v1
+
+    try:
+        from ..models import apiextensions_v1 as apiextensions
+    except:
+        from ..models import apiextensions_v1beta1 as apiextensions
```

Comment:
In case of import failure we need to define a mock for each method/class used in apiextensions_v1. See bottom of this same file for examples (copied over comment as it was not visible any longer).

Reply:
Just FYI, you may get a conflict on this file as #37 was merged.